README.md

```diff
@@ -18,6 +18,7 @@ Want to go into deep? Please checkout the [documentation](docs).
 ### Install it as a Helm Chart
 
 First, please clone this git repository. Then run command: `make install-chart`
+ - Because the default registry uses `ghcr.io/kubesphere-sigs`, if you want to use `docker.io` as the registry for image pull, you can modify `chart/ks-devops/values.yaml image.registry`
```

Comment:
It's incorrect. People don't need to modify anything. They just need to pass parameters into the helm command. For example:

`helm install ks-devops . --set image.registry=docker.io`

Reply:
or how to add parameters in `make install-chart`

Reply:
There're many parameters in the helm chart. Normally, all the descriptions should be able to find in the readme file of the helm chart.

![image](https://user-images.githubusercontent.com/1450685/126594702-5d5c1d98-e611-4f31-be41-1d97b2146794.png)

And in most use cases, people install helm charts by the helm command instead of the make command. So I don't recommend do more things on the make command. The make command is only useful for the developers.

Reply:
OK